tendrl/commons/objects/definition/master.yaml

```diff
@@ -83,11 +83,14 @@ namespace.tendrl:
 
   objects:
     Cluster:
-      internal: True
       enabled: True
       help: "Represents a cluster"
       list: /clusters
+      value: clusters/$TendrlContext.integration_id
       attrs:
+        integration_id:
+          help: integration id
+          type: String
```

Comment:
Why `list` and how we would call load() in that case? Currently load is something like

```
NS.tendrl.objects.Cluster(integration_id=NS.tendrl_context.integration_id).load()
```

How this would work in case we have identifier as `list`?